.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: ruby
 sudo: false
 cache: bundler
-bundler_args: --without benchmarks tools
+bundler_args: --without benchmarks
 script:
   - bundle exec rake spec
+  - bundle exec rubocop
```

Comment:
This has room for improvement since it now runs rubocop in scope of every matrix build but I'm not sure how exactly to set it up to run in a build of its own. I can revisit that later as now it is not a big problem, rubocop runs for just 3-4s.